frontend/src/components/RequestRideModal/RequestRideInfo.tsx

```diff
@@ -10,9 +10,12 @@ import CustomRepeatingRides from './CustomRepeatingRides';
 
 type RequestRideInfoProps = {
   ride?: Ride;
+  showRepeatingCheckbox: boolean;
+  showRepeatingInfo: boolean;
 }
 
-const RequestRideInfo = ({ ride }: RequestRideInfoProps) => {
+const RequestRideInfo = ({ ride, showRepeatingCheckbox,
+  showRepeatingInfo }: RequestRideInfoProps) => {
```

Comment:
when would we want the repeating checkbox but not the repeating info (or vice versa)?

Reply:
think this relates to the bug im seeing

Reply:
The repeating checkbox only shows up when creating a ride, not when editing a ride, and repeating info shows up for both if it's a repeating ride--I'll look into the bug!